PublishParasoftResults/ParaReportPublishService.ts

```diff
@@ -203,9 +203,13 @@ export class ParaReportPublishService {
     }
 
     run = async (): Promise<void> => {
-        // Clean up the old custom markdown summary storage directory before executing subsequent quality gate tasks
-        tl.rmRF(tl.resolve(this.defaultWorkingDirectory, 'ParasoftQualityGatesMD'));
-
+        // Check if there are multiple "Publish Parasoft Results" tasks in the pipeline to prevent confusion
+        const publishTaskExists = tl.getVariable('PF.PublishParasoftResultsExists');
+        if (!publishTaskExists) {
+            // Clean up the old custom markdown summary storage directory when first run the PublishParasoftResults task
```

Comment:
Is it better to chang the comment to 'Clean up the old custom markdown summary storage directory when running the first PublishParasoftResults task'.